roles/container-engine/docker/tasks/main.yml

```diff
@@ -37,16 +37,6 @@
   tags:
     - facts
 
-- name: check for minimum kernel version
-  fail:
-    msg: >
-          docker requires a minimum kernel version of
-          {{ docker_kernel_min_version }} on
-          {{ ansible_distribution }}-{{ ansible_distribution_version }}
-  when: (not ansible_os_family in ["Flatcar Container Linux by Kinvolk", "ClearLinux"]) and (ansible_kernel is version(docker_kernel_min_version, "<"))
-  tags:
-    - facts
-
```

Comment:
Why can we remove this check?
The oldest docker_kernel_min_version seems kernel v3.10.
Can we say kernel version should be always newer than it?

Reply:
Only CentOS 7 ship a kernel that old (with a lot of backports), all other distro ship something newer than 3.10 for years
Ubuntu 16.04 == Linux 4.4
Debian 9 == Linux 4.9

Reply:
Thanks for your explanation. It seems fine to remove here.